docs/deployment/configuring_logging.md

```diff
@@ -80,3 +80,23 @@ logging:
   loki:
     enabled: false
 ```
+
+# Configuring central Loki storage capacity
+
+By default central Loki has 100Gi of storage capacity. 
+To overwrite the current central Loki storage capacity the `logging.loki.garden.storage` setting should be altered.
+If we wish to increase it we can do it without loosing the current data by specifying higher capacity. Doing so the Loki PersistentVolume capacity will be increased instead of deleting the current PV.
```

Comment:
```suggestion
If you need to increase it you can do so without loosing the current data by specifying higher capacity. Doing so, the Loki's `PersistentVolume` capacity will be increased instead of deleting the current PV.
```